.github/workflows/sdk-lint.yaml

```diff
@@ -0,0 +1,36 @@
+name: SDK Lint
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Setup Gradle cache
+        uses: actions/cache@v2.1.5
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Run detekt task
+        run: ./gradlew --build-cache --no-daemon --info detekt
+      - name: Upload SARIF files
+        uses: github/codeql-action/upload-sarif@v1
+        if: ${{ always() }}
+        with:
+          sarif_file: .
```

Comment:
The tool can only use a single directory/file as input and will recursively search for any `.sarif` file. Using `.` will use the project directory to search in, this will search all directories in the project which might be slow (relatively speaking) but it will at least find all sarif files, even if we add more tools.